docs/cli-guide/master.adoc

```diff
@@ -69,7 +69,10 @@ include::topics/get-involved.adoc[leveloffset=+3]
 include::topics/important-links.adoc[leveloffset=+3]
 
 // Report Issues with {ProductShortName}
-include::topics/report-issues.adoc[leveloffset=+3]
+
+==== Reporting issues
+
+{ProductShortName} uses Jira as its issue tracking system. If you encounter an issue executing {ProductShortName}, create a Jira issue by opening a browser and navigating to the link:https://issues.redhat.com/projects/WINDUP[Create issue] page.
```

Comment:
"create a Jira issue by opening a browser and navigating to the link:https://issues.redhat.com/projects/WINDUP[Create issue] page."

Change globally to this:

"submit a link:https://issues.redhat.com/projects/WINDUP[Jira issue]."

As I mentioned in an earlier comment, it is a Jira issues page, not a Jira "Create issue" page.
No need to tell the user to open a browser if you are providing a hyperlink. Most users are aware of this.